connectivity/FEATURE_BLE/source/pal/PalGap.h

```diff
@@ -183,6 +185,7 @@ struct PalGapEventHandler {
      */
     virtual void on_legacy_advertising_stopped() = 0;
 
+#if BLE_FEATURE_EXTENDED_ADVERTISING
```

Comment:
don't these work for legacy handle as well?

Reply:
There's `on_legacy_advertising_started` and `on_legacy_advertising_stopped` for legacy advertising. There is no HCI event signaling scab request received before BT 5